packages/net/_test.pony

```diff
@@ -282,16 +282,24 @@ class _TestTCPExpectNotify is TCPConnectionNotify
 
   fun ref accepted(conn: TCPConnection ref) =>
     conn.set_nodelay(true)
-    conn.expect(_expect)
-    _send(conn, "hi there")
+    try
+      conn.expect(_expect)?
+      _send(conn, "hi there")
+    else
+      conn.close()
```

Comment:
If hitting the `else` here is an unexpected condition, should we mark a failure of the test rather than just closing the connection?

Same question with the other tests that were changed in this PR.

Reply:
interesting. yes. we should. i was thinking about it from the perspective of "not a test".